### PR TITLE
Added fix for m/s to mph and isNight to weather object

### PR DESCRIPTION
**Pull request by napalmcsr**
I use isNight to update the image.

### CODE REVIEW

File: usr/bin/js/openWeather.js [hunk 1]
```diff
@@ -74,11 +76,17 @@ function getOpenWeather (location, tempUnit, distanceUnit) {
             else {
                 weather='cloudy';
             }
+	var datenow = new Date();
+	var timenow = Math.floor(datenow.getTime()/1000);
+	var isNight = (timenow < jsonResult.sys.sunrise || timenow > jsonResult.sys.sunset);
+
```
Comment (umamiking) [inline usr/bin/js/openWeather.js]: Not to be too nitpicky but can we avoid a lot of unnecessary new lines?

Comment (napalmcsr) [reply usr/bin/js/openWeather.js]: I take suggestions on how you would like to see it?

File: usr/bin/js/openWeather.js [hunk 2]
```diff
@@ -74,11 +76,17 @@ function getOpenWeather (location, tempUnit, distanceUnit) {
...
+
 
             var weatherObj = {
                 'temperature': tempVal,
                 'condition': weather,
-                'wind': windSpeed
+                'wind': windSpeed,
+		'isNight': isNight
```
Comment (ouellettetech) [inline usr/bin/js/openWeather.js]: also can you align the properties. I probably should have also aligned the values...

**Review by ouellettetech — CHANGES_REQUESTED** [review]

File: usr/bin/js/openWeather.js (hunk 2 above)
Comment (napalmcsr) [reply usr/bin/js/openWeather.js]: I think I updated it.

File: usr/bin/js/openWeather.js (hunk 1 above)
Comment (napalmcsr) [reply usr/bin/js/openWeather.js]: removed newline, but added a comment

**Comment by ouellettetech** [comment]
I'm going to let it through, and then fix the math on the kph, mph

**Pull request merged**